File: Strings/Java/PangramChecker.java
// A string is a pangram if it contains all 26 letters.

public class PangramChecker {
    public static void main(String[] args) {
        String str = "The quick brown fox jumps over the lazy dog".toLowerCase();
        boolean[] present = new boolean[26];
        for (char c : str.toCharArray())
            if (Character.isLetter(c)) present[c - 'a'] = true;
        for (boolean b : present) {
            if (!b) {
                System.out.println("Not Pangram");
                return;
            }
        }
        System.out.println("Pangram");
    }
}

File: Strings/Python/pangram_checker.py
def is_pangram(s):
    s = s.lower()
    return all(chr(c) in s for c in range(ord('a'), ord('z')+1))
    #return len(set(filter(str.isalpha, s.lower()))) == 26

print("Pangram" if is_pangram("The quick brown fox jumps over the lazy dog") else "Not Pangram")

# A string is a pangram if it contains all 26 letters.